packages/salesforcedx-visualforce-language-server/src/modes/javascriptMode.ts

```diff
@@ -133,7 +133,8 @@ export function getJavascriptMode(
       const offset = currentTextDocument.offsetAt(position);
       const completions = jsLanguageService.getCompletionsAtPosition(
         FILE_NAME,
-        offset
+        offset,
+        { includeExternalModuleExports: false }
```

Comment:
We now need to explicitly fill in the arguments (can't call a method that expects x arguments with y arguments, JavaScript style). See TS2554 for the error.